staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -274,6 +276,7 @@ func (flags *ApplyFlags) ToOptions(cmd *cobra.Command, baseName string, args []s
 	}
 
 	openAPISchema, _ := flags.Factory.OpenAPISchema()
+	openapiGetter := flags.Factory.OpenAPIGetter()
```

Comment:
> Actually, I see, they're not the same type. One possibility would be to wire the information you care about into the `openAPISchema` above, to make it readily accessible.

Good idea. Thanks to that, patcher only manages `openAPISchema`.

Reply:
> It's a little painful because the code is in `k8s.io/kube-openapi` but that should be ok.

Does that mean can I open a PR in kube-openapi adding this? :)

Reply:
Yeah, yeah, go ahead!